readAndGetContent.cpp

```diff
@@ -76,8 +76,8 @@ void ProcessManager::getContent(string Module)
 			}
 			s = "0x" + s;
 			contentLongInt = stol(s, NULL, 16);
-			cout<<dec<<contentLongInt<<endl;//Comment Line When Not Debugging
-			cout <<hex<< contentLongInt << endl;//Comment Line When Not Debugging
+			// cout<<dec<<contentLongInt<<endl;//Comment Line When Not Debugging
+			// cout <<hex<< contentLongInt << endl;//Comment Line When Not Debugging
```

Comment:
What I would recommend is having an option when I run the binary. 
Kind of like ./mem --debug or ./mem --verbose=2 (the level of logging that needs to be done).
This would mean we don't have to focus on what is commented and what is not.

Reply:
Okay, I'll go through that.